pulsar-client-cpp/include/pulsar/MessageBuilder.h

```diff
@@ -71,6 +71,11 @@ class MessageBuilder {
     MessageBuilder& setPartitionKey(const std::string& partitionKey);
 
     /**
+     * Set the event timestamp for the message.
+     */
+    MessageBuilder& setEventTimestamp(uint64_t event_timestamp);
```

Comment:
Nitpick: in C++ code we've been using camelCase for classes and var names, like `eventTimestamp`